config/default.yml

```
@@ -325,6 +325,11 @@ Rails/LinkToBlank:
   Enabled: true
   VersionAdded: '0.62'
 
+Rails/NegateInclude:
+  Description: 'Prefer `collection.exclude?(obj)` over `!collection.include?(obj)`.'
+  Enabled: true
```

RuboCop Rails would like to introduce the pending status from the next minor release. Can you update this status to `'pending'`?

```suggestion
  Enabled: 'pending'
```